programs/mango-v4/src/state/orderbook/book.rs

```diff
@@ -138,12 +138,13 @@ impl<'a> Orderbook<'a> {
                 matched_order_changes.push((best_opposing.handle, new_best_opposing_quantity));
             }
 
+            let seq_num = event_queue.header.seq_num;
```

Comment:
this change is baffling to me, because my version of book.rs already has it applied. I don't see why github shows it again

Reply:
User error on my part - now resolved.